README.md

```diff
@@ -7,25 +7,42 @@ Latest build: https://metamask.github.io/api-specs/latest/openrpc.json
 
 JSON-RPC API provided by MetaMask.
 
-MetaMask dapp developers lack a formal RPC specification. This attempts to document which methods MetaMask supports in a way that's versionable, human-readable and machine-readable. It will improve the accuracy of documentation, API, and clients.
+Previously, MetaMask dapp developers lacked a formal RPC specification. This repository contains a specification that documents which methods MetaMask supports in a way that's versionable, human-readable, and machine-readable. This spec improves the accuracy of documentation, API, and clients.
 
-MetaMasks effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
+MetaMask's effectiveness as an enterprise-grade application platform depends on its ability to provide a reliable and predictable developer experience.
 
 ## Contributing
 
-Edit the `openrpc.yaml` file.
+> **Note:** To make a significant change to the API, please submit a [MIP](https://github.com/MetaMask/metamask-improvement-proposals/tree/main).
 
-You can then run `yarn run build` to generate the output file again.
-
-To test the result, you can paste the resulting file from `dist/build/openrpc.json` into [the OpenRPC playground](https://playground.open-rpc.org/) and view the result.
+1. Edit the API specification in the `openrpc.yaml` file.
+2. Run `yarn install` if you haven't previously set up the repository. 
+3. Run `yarn run build` to re-generate the output file: `dist/build/openrpc.json`.
+4. To view the result, paste that file's contents into [the OpenRPC playground](https://playground.open-rpc.org/).
 
 ### Build Process
-- filter out methods that are not supported/implemented by metamask
-- merge wallet methods with execution api methods
-- output to `build`
 
+The build process:
+- filters out methods that are not supported/implemented by MetaMask
+- merges wallet methods with execution API methods
+- outputs to `dist` for release
+
+<details>
+  <summary>Build and merge summary</summary>
```

Comment:
```suggestion
```

I prefer to remove this first list and just keep the following more detailed list to reduce redundancy. We can also take it out of the dropdown in this case.

Reply:
OK -- I prefer the get a short version and expand to find out more approach, but it's not a hill I am keen to die on